packages/web/lib/networking/mutations/unsubscribeMutation.ts

```diff
@@ -12,11 +12,12 @@ type Unsubscribe = {
 }
 
 export async function unsubscribeMutation(
-  subscribeName: string
+  subscribeName: string,
+  id = ''
 ): Promise<any | undefined> {
   const mutation = gql`
     mutation {
-      unsubscribe(name: "${subscribeName}") {
+      unsubscribe(name: "${subscribeName}", subscriptionId: "${id}") {
```

Comment:
Better to use `input` for variables if possible

Reply:
Got it, yeah let's not change existing mutations in this PR. I can do that separately.